node/clients/search-graphql.ts

```diff
@@ -1,18 +1,22 @@
-import { AppGraphQLClient, IOContext, InstanceOptions } from "@vtex/api";
+import { IOContext, InstanceOptions } from "@vtex/api";
+import { GraphQLServer } from './graphql-server';
+
 import { pathOr } from "ramda";
 
-export class SearchResolver extends AppGraphQLClient {
+const extensions = {
```

Comment:
graphql-server client could export a function to generate these extensions, what do you think ?

Reply:
awsome! I will do it